### DISCUSSION TITLE
Found in Build.md

### DISCUSSION THREAD

**Discussion by sandstone42** [GrandOrgue/grandorgue#864] [Developers] [answered]
In the build instructions for compiling wxWidgets there was a line, "git apply /gosrc/wxWidgets/wxWidgets-fix-build-3.0.2.patch"
however, there is no wxWidgets directory in the source code. Is this leftover from something else?

**Comment by oleg68** [comment, marked as answer]
The build instruction for windows is obsolete.

Now the only supported way of building for windows is ``Simplified cross-compiling for Windows on Linux``
